### ISSUE TITLE
Documentation on axios-http.com is outdated

### ISSUE THREAD

**Issue by Steel-Shadow** [axios/axios#6398]
### Section/Content To Improve

[Request Config documentation on axios-http.com](https://axios-http.com/docs/req_config) is outdated.

### Suggested Improvement

`withXSRFToken` is missing from the [documentation above](https://axios-http.com/docs/req_config).
The README.md on the github repo is up to date. But the documentation on axios-http.com is not.

The following is in README.md but not on the documentation site.
```
 // `undefined` (default) - set XSRF header only for the same origin requests
  withXSRFToken: boolean | undefined | ((config: InternalAxiosRequestConfig) => boolean | undefined),
```

The website should be updated in time.

**Comment by sixmachine-6** [comment]
i will try

**Comment by jasonsaayman** [comment]
yeah i think it may be time to make a change to that.... i think the docs not living next to the code really makes it a chore for contributors to work on the docs... i will have a look at improving here and making things better

**Comment by sixmachine-6** [comment]
how should i approach sir...

**Comment by jasonsaayman** [comment]
you dont need to, i am half way through making new docs and I will then add them to the repo

**Comment by jasonsaayman** [comment]
for now we will need to try sync the docs and this is something i am looking into.
